app.js

```diff
@@ -0,0 +1,17 @@
+const { getTopics } = require("./controllers/topics.controller.js");
+const express = require("express");
+const res = require("express/lib/response");
```

Comment:
This line isn't needed! Think it's probably auto-imported.(You can just change it with the PR for your next ticket if you like.)